sql/executor.go

```diff
@@ -206,8 +206,10 @@ func (e Executor) execStmt(stmt parser.Statement, params parameters, planMaker *
 						row.Values = append(row.Values, driver.Datum{
 							Payload: &driver.Datum_IntervalVal{IntervalVal: vt.Nanoseconds()},
 						})
+					case parser.DTuple:
+						return fmt.Errorf("unsupported result type: tuple, remove extra parenthesis around column %s", result.Columns[i])
```

Comment:
@vivekmenezes you can get "tuple" instead of "parser.DTuple" by calling `Type()` on the datum.